ext/vl53/vl53_transport.hpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2025, Henrik Hose
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_VL53_TRANSPORT_HPP
+#define MODM_VL53_TRANSPORT_HPP
+
+#include <array>
+#include <cstdint>
+#include <modm/architecture/interface/delay.hpp>
+#include <modm/architecture/interface/i2c_device.hpp>
+#include <modm/architecture/interface/spi_device.hpp>
+#include <modm/processing/fiber.hpp>
+#include <optional>
+#include <span>
+
+namespace modm
+{
+
+class Vl53TransportBase
+{
+public:
+	virtual ~Vl53TransportBase() = default;
+
+	virtual uint8_t
+	writeMulti(uint16_t reg, uint8_t *values, uint32_t size) = 0;
```

Comment:
```suggestion
	writeMulti(uint16_t reg, const uint8_t *values, uint32_t size) = 0;
```

It doesn't hurt to make it `const`-correct even if the calling code doesn't ask for it.

Reply:
fixed